README.textile

```diff
@@ -1,5 +1,9 @@
 h2. Please refer to "attributerouting.net":http://attributerouting.net/ for documentation.
```

Comment:
I put the pieces together from the inserted text below. Can you just add a link to `http://htmlpreview.github.io/?https://github.com/mccalltd/AttributeRouting/blob/gh-pages/index.html`?